zilencer/views.py

```diff
@@ -106,3 +114,66 @@ def remote_server_notify_push(request,  # type: HttpRequest
                                      badge=1, zulip=apns_payload)
 
     return json_success()
+
+def register_remote_server(request):
+    # type: (HttpRequest) -> HttpResponse
+    if request.method == 'POST':
+        form = RemoteServerRegistrationForm(request.POST)
+        if form.is_valid():
+            email = form.cleaned_data['email']
+            api_key = form.cleaned_data['api_key']
+            uuid = form.cleaned_data['uuid']
+            hostname = form.cleaned_data['hostname']
+            with transaction.atomic():
+                status_obj = RemoteServerRegistrationStatus.objects.create(
+                    email=email,
+                    hostname=hostname,
+                    uuid=uuid,
+                    api_key=api_key,
+                )
+                template_prefix = 'zilencer/emails/confirm_remoteserver'
+                context = {'hostname': hostname}
+                RemoteServerConfirmation.objects.send_confirmation(
```

Comment:
I think you can just call send_email directly, since there is only one email.
There's some documentation on the new email system at http://zulip.readthedocs.io/en/latest/email.html